code-challenges.js

```diff
@@ -6,55 +6,159 @@
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
 // Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
+//create a function called testTemp that takes in one parameter called temp
+//create a condition that checks if temp is below boiling point or above boiling point
+//make variable called boilingPoint
+//if below boilingPoint return `temp is below boiling point"
+//if above boiling Point return `temp is above boiling point"
+//if temp is === 212 return "temp is at boiling point"
+
 var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
-
-
-
+// const testTemp = (temp) => {
+//     let boilingPoint = 212;
+//     if (temp < boilingPoint) {
+//         return `${temp} is below boiling point`;
+//         }
+//     else if (temp > boilingPoint) {
+//         return `${temp} is above boiling point`;
+//         }
+//         else {
+//             return `${temp} is at boiling point`
+//         }
+//     }
+
+//tried ternary operators for experience and shorter code/cleaner? 
+const testTemp = (temp) => {
+    let boilingPoint = 212;
+    let result = "";
+
+return result = temp < boilingPoint ? `${temp} is below boiling point` : (temp > boilingPoint ? `${temp} is above boiling point` : `${temp} is at boiling point`)
+    }
```

Comment:
some small spacing and indentations with this one. you can also omit the parentheses in the return of this solution :) 

nice work using a ternary! i love them, they are so cool!